spec/cedar_policy/authorizer_spec.rb

```
@@ -172,5 +172,152 @@
+          [
+            {uid: {type: "User", id: "1"}, attrs: {}, parents: []},
+            {uid: {type: "User", id: "2"}, attrs: {isAdmin: true}, parents: []},
+            {uid: {type: "Image", id: "1"}, attrs: {owner: {type: "User", id: "1"}}, parents: []}
+          ]
+        }
+
+        it { is_expected.to(have_attributes(decision: CedarPolicy::Decision::ALLOW)) }
+      end
+
+      describe "entity with superfluous attribute" do
+        let(:entities_data) {
```

rubocop (1.68.0)

[convention] Avoid using `{...}` for multi-line blocks.

(Style/BlockDelimiters)